src/templates/video.js

```diff
@@ -1,55 +1,11 @@
 import React from "react"
```

Comment:
I'm not sure if or where this template is used. Don't merge prior to figuring that out